examples/lighting-app/stm32/README.md

```diff
@@ -17,10 +17,18 @@ Please refer
 [Building and Commissioning](../../../docs/guides/stm32_getting_started_guide.md#building-the-example-application)
 guides to get started
 
+Note: the following command from the original repository is not correct. The correct one is as followed.
 ```
+// this command is not working, -lighting shall be -light
```

Comment:
Why are we leaving non-working stuff in here?  Please let's just have the working thing....  Then the Note: is not needed either, right?